### DISCUSSION TITLE
Why is my "default" color name not working for border colors?

### DISCUSSION THREAD

**Discussion by audn** [tailwindlabs/tailwindcss#2226] [Help] [answered]
Hey, I'm trying to get my customized borders to work, but it seems like they don't work when my customized color's property starts with "Default".

```js
module.exports = {
  rarity: {
    rare: '#00EA5C',
  },
  default: {
    rare: '#00EA5C',
  },
};
```
Above I have two properties, one for default colors, and one for rarities.

Using the rarity property
```html
<div className="bg-white border-4 border-rarity-rare">
    border-rarity-rare
</div>
```
![Using rarity-rare](https://user-images.githubusercontent.com/21041399/90959981-a4daf480-e49e-11ea-98ae-ed07eac0e3e1.png)

Using the default property
```html
<div className="bg-white border-4 border-default-rare">
    border-default-rare
</div>
```
![Using default-rare](https://user-images.githubusercontent.com/21041399/90959974-98ef3280-e49e-11ea-9958-acb5cb5a959f.png)

Using the `default-rare` as a background color works. It only seems like this is an issue with border. 
What am I doing wrong?

**Comment by Apostolique** [comment]
I didn't test this, but does using `border-rare` work?

**Reply by audn** [reply]
Nice one! But no :(

**Comment by audn** [comment]
What's the chances of receiving help in here?

**Comment by adamwathan** [comment, marked as answer]
This is a limitation by design currently because `default` is reserved for setting the default border color on all elements. You'll notice we don't generate `border-default` for example.

Unfortunately for now the workaround is to simply choose a different word.
